web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx

```diff
@@ -36,6 +36,7 @@ export function IntegrationsAddButton({
 
   return (
     <HoverTooltip
+      position="bottom"
```

Comment:
No `<FeatureHeader justifyContent="space-between">` in v16?

Reply:
Looking into it now

Reply:
Oh, this doesn't need to go to v16. I forgot I [fixed it in that backport](https://github.com/gravitational/teleport/pull/50246/files#diff-cc99c882ccccd7acee329b7e5f63ef2935c0e306bcfbe87c44c523fbb7611731R82) due to other conflicts. I'll close this

Reply:
@avatus FWIW, v16 doesn't have `position="bottom"`.

https://github.com/gravitational/teleport/blob/970f4f9ae603196a92263dbd724081e2cf33fc3e/web/packages/teleport/src/Integrations/IntegrationsAddButton.tsx#L38-L48

Reply:
I'm going to ask around to understand why the recent changes to `HoverTooltip` didn't make it back to v16